nixos/modules/services/mail/rss2email.nix

```diff
@@ -108,15 +110,16 @@ in {
     in
     {
       preStart = ''
-        cp ${conf} /var/rss2email/conf.cfg
-        if [ ! -f /var/rss2email/db.json ]; then
-          echo '{"version":2,"feeds":[]}' > /var/rss2email/db.json
+        cp -f ${conf} /var/lib/rss2email/conf.cfg
```

Comment:
does rss2email mutate its conf file by itself, or why is this mutable?

Reply:
Unfortunately yes, it needs to be able to mutate its configuration at runtime, otherwise it crashes. it doesn't need the changes to be persistent across runs, though, so `cp`'ing it in-place just works

Reply:
And other distros haven't complained about that? If I look at a standard `debian` install everything under `/etc` is `-rw-r--r--` with `root:root`. Allowing unprivileged users to permanently modify configuration like that seems like a (minor) security risk on a "regular" (non NixOS) system :thinking:

Reply:
Yeah, if I actually understood the code well I'd probably try to remove that requirement, but… I guess for now the concept is to just `chmod` the configuration file as non-root and point `rss2email` to it (or even, run it manually, as appears to be done in quite a few cases)

Reply:
Can we open an issue upstream. to raise awareness on that?

Reply:
Just done at https://github.com/rss2email/rss2email/issues/110 ; but with the current maintenance status, I wouldn't raise too high hopes for a timely resolution, unless someone drops by and submits a PR.

Reply:
alright - but this shouldn't block the PR.